pkg/rsnotify/listeners/postgrespgx/pgxlistener.go

```diff
@@ -224,13 +224,17 @@ func (l *PgxListener) notify(n *pgconn.Notification, errs chan error, items chan
 		errs <- fmt.Errorf("error unmarshalling message data type: %s", err)
 		return
 	}
-	if l.matcher.Type(dataType) == nil {
+	t, err := l.matcher.Type(dataType)
+	if err != nil {
+		return
+	}
+	if t == nil {
```

Comment:
If you want to be able to ignore errors, you'll need to remove this block. I think we probably want this behavior in the generic matcher. Then, using the generic matcher will return this error when no type is found, but we can create a custom matcher for offline mode that returns no error.

Reply:
oops. 
I did the opposite: genericMatcher return no error (no changes in the GenericMatcher) and OfflineMatcher return `MissingType` error.

What kind of interface should OfflineMatcher return in case of missing type?

Reply:
I was thinking that the generic matcher would return the error on a missing type, while the offline matcher would simply return a nil error. Then, in the listener implementations, if we get back a nil value and no error, we would simply return with no error.